src/app.ts

```diff
@@ -1,6 +1,22 @@
 
 const startApp = async () => {
-    
+    const list = await fetch('https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84')
+    .then(
+        result => result.json()
+    )
+    console.log({list});
```

Comment:
Codacy has a fix for the issue: [Replace `····console.log({list` with `··console.log({·list·`](https://app.codacy.com/gh/ZeeUnltd/starter/pullRequest?prid=10061291)
```suggestion
  console.log({ list });
```